requests-handler.ts

```diff
@@ -0,0 +1,133 @@
+import {DefaultContext, ExtendableContext} from 'koa';
```

Comment:
this file is a controller - should be called `products-controller` and be inside `controllers` folder

Reply:
done